policyengine/countries/uk/reform_parameters.yaml

```
@@ -76,6 +76,24 @@ reforms:
           description: Age at which a person receives the working-age adult UBI payment.
           max: 65
           summary: Set the working-age UBI age threshold to @
+    taxable:
+      values: 
+        2021-01-01: false
+      metadata:
+        policyengine:
+          title: Taxable UBI
+          short_name: taxable_UBI
+          type: bool
+          description: Whether the UBI is included in taxable income for Income Tax.
+    in_means_tests:
+      values: 
+        2021-01-01: false
+      metadata:
+        policyengine:
+          title: Included in means tests
```

```suggestion
          title: UBI included in means tests
```
For consistency. Could also be just `Taxability` / `Included in means tests`.

---

Thanks, added